service_container.rst

```diff
@@ -503,12 +502,19 @@ and reference it with the ``%parameter_name%`` syntax:
     .. code-block:: php
 
         // config/services.php
+        namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
         use App\Updates\SiteUpdateManager;
-        $container->setParameter('admin_email', 'manager@example.com');
 
-        $container->autowire(SiteUpdateManager::class)
-            // ...
-            ->setArgument('$adminEmail', '%admin_email%');
+        return function(ContainerConfigurator $configurator) {
+            $configurator->parameters()->set('admin_email', 'manager@example.com');
+
+            $container = $configurator->services();
+            $container->set(SiteUpdateManager::class)
+                ->autowire()
```

Comment:
To be consistent with other examples you should add an `// ...` before and remove `->autowire()`

Reply:
Good catch, i'll go through and make those more consistent, I thought I had done that, but I obviously missed some.